src/main/kotlin/org/kamiblue/client/command/commands/SetCommand.kt

```diff
@@ -90,15 +105,21 @@ object SetCommand : ClientCommand(
                 val module = moduleArg.value
                 val settingList = module.fullSettingList
 
-                MessageSendHelper.sendChatMessage("List of settings for ${formatValue(module.name)}: " +
-                    formatValue(settingList.size)
-                )
-                MessageSendHelper.sendRawChatMessage(settingList.joinToString { it.name })
+                MessageSendHelper.sendChatMessage("List of settings for ${formatValue(module.name)}: ${formatValue(settingList.size)}")
```

Comment:
I don't think the setting list size should be including fullSettingList, which has the module enabled state and visible state, and other stuff

Reply:
I don't really know why I added that neither. 🗿

Reply:
I mean setting list size is fine, but it should be normal setting list, not full

Reply:
> I don't really know why I added that neither. 🗿

Uhh replied to the wrong one. Was for "Why are we printing settings with lowercase in the chat? It's less readable"

Reply:
> I don't think the setting list size should be including fullSettingList, which has the module enabled state and visible state, and other stuff

No, set command should allow you change all setting including visible, invisible ones. And it is only 3 or 4 settings

Reply:
Alright fine. Just the lowercase when printing is dumb